std/c/freebsd.zig

```diff
@@ -1,4 +1,8 @@
+const std = @import("../std.zig");
+use std.c;
+
 extern "c" fn __error() *c_int;
 pub const _errno = __error;
 
 pub extern "c" fn getdents(fd: c_int, buf_ptr: [*]u8, nbytes: usize) usize;
+pub extern "c" fn sigaltstack(ss: ?*stack_t, old_ss: ?*stack_t) c_int;
```

Comment:
Wait what? It's *sig*nal *alt*ernate *stack* also in [FreeBSD](https://www.freebsd.org/cgi/man.cgi?query=sigaltstack&sektion=2&apropos=0&manpath=freebsd) [OpenBSD](https://man.openbsd.org/sigaltstack.2) and [Linux](http://man7.org/linux/man-pages/man2/sigaltstack.2.html)

Reply:
Ah, my mistake. I saw `/home/vsts/work/1/s/std/os.zig:2458:25: error: container 'c' has no member called 'sigaltstack'` and made an incorrect assumption.

So this looks like a bug with use then? damn

Reply:
> So this looks like a bug with `use` then? damn

ah right of course. it was just the same darwin bug on all 3 of the CI because they each test cross compilation.

Reply:
> So this looks like a bug with use then? damn

Nope, I just forgot to add the declaration in `darwin.zig`

Reply:
well I feel silly. sorry for all the noise